tensorflow_lite_support/ios/task/vision/utils/sources/GMLImage+Utils.m

```diff
@@ -265,17 +247,13 @@ + (UInt8 *_Nullable)pixelDataFromCGImage:(CGImageRef)cgImage error:(NSError **)e
   return buffer_to_return;
 }
 
-- (TfLiteFrameBuffer *)frameBufferFromCGImage:(CGImageRef)cgImage error:(NSError **)error {
-  UInt8 *buffer = [UIImage pixelDataFromCGImage:cgImage error:error];
+- (uint8_t *)frameBufferFromCGImage:(CGImageRef)cgImage error:(NSError **)error {
+  uint8_t *buffer = [UIImage pixelDataFromCGImage:cgImage error:error];
 
-  return [TFLCVPixelBufferUtils cFrameBufferWithWidth:(int)CGImageGetWidth(cgImage)
-                                               height:(int)CGImageGetHeight(cgImage)
-                                    frameBufferFormat:kRGB
-                                               buffer:buffer
-                                                error:error];
+  return buffer;
 }
 
-- (TfLiteFrameBuffer *)frameBufferFromCIImage:(CIImage *)ciImage error:(NSError **)error {
+- (uint8_t *)frameBufferFromCIImage:(CIImage *)ciImage error:(NSError **)error {
   uint8_t *buffer = NULL;
 
   int width = 0;
```

Comment:
I think width and height aren't necessary here anymore.

Reply:
Fixed